Sources/Playdate/Documentation.docc/BuildingTheExamples.md

```diff
@@ -93,3 +99,60 @@ Select the Run icon in the toolbar or press Cmd+R on your keyboard to start a bu
 
 > Note:
 > Learn more about [Xcode](https://developer.apple.com/xcode/)
+
+
+## Determining the toolchain
+
+When using `make` to build the examples, the latest installed Swift toolchain will be automatically determined. It may, however, sometimes be useful to specify a specific toolchain version. It is always necessary to define the toolchain when building with swift package manager in terminal.
+
+> Note: When using `make` to build the examples, Swift toolchains installed for the current user take precedence over globally installed toolchains.
+
+1. The toolchain identifier can be found under the `CFBundleIdentifier` key in the `Info.plist` file found at the root of the toolchain.
+
+You can retrieve the toolchain's identifier by running the following command in your terminal:
+```console
+$ plutil -extract CFBundleIdentifier raw -o - /Library/Developer/Toolchains/swift-latest.xctoolchain/Info.plist
+org.swift.59202312211a
+```
+
+@Image(source: "swift-toolchain-version", alt: "A screenshot of a Terminal application showing the result of the plutil command.")
+
+> Note: If you selected **Install for me only** rather than **Install for all users on this computer**, you can retrieve the toolchain's identifier by running the following command in your terminal:
+> ```console
+> $ plutil -extract CFBundleIdentifier raw -o - ~/Library/Developer/Toolchains/swift-latest.xctoolchain/Info.plist
+> org.swift.59202312211a
+> ```
+> 
+> Or use a single command to determine the identifier of the latest toolchain(s) in case you have both local as well as global installations:
+> ```console
+> $ find ~/Library/Developer/Toolchains /Library/Developer/Toolchains -name "swift-latest.xctoolchain" -depth 1 -exec plutil -extract CFBundleIdentifier raw -o - {}/Info.plist \; 2>/dev/null|sort
+>```
+
+2. Set `TOOLCHAINS` as an environment variable when building with `make` or `swift build`, but this will not work when building with a graphical editor.
+
+```console
+$ TOOLCHAINS=org.swift.59202403121a make
+```
+
+```console
+$ TOOLCHAINS="org.swift.59202312211a" swift build -c release
+```
+
+> Note: You can alternatively specify your toolchain name in the `Examples/swift.mk` file. Edit `swift.mk` in the `Examples` directory to set `TOOLCHAINS := "<your Swift nightly toolchain name>"` before the `TOOLCHAINS` check. The following example diff uses the name `"org.swift.59202312211a"`.
```

Comment:
Lets make this its own section instead of a Note

Reply:
It now looks like this:
<img width="742" alt="image" src="https://github.com/apple/swift-playdate-examples/assets/1049693/13c88427-b65f-4e7e-af1b-11281029e3ae">
<img width="748" alt="image" src="https://github.com/apple/swift-playdate-examples/assets/1049693/12851aef-0309-4257-ad9d-c0463116e961">